Source/sdc/sdc_util.cpp

```diff
@@ -415,14 +415,20 @@ void Castro::ca_store_reaction_state(const Box& bx,
 
     AMREX_PARALLEL_FOR_4D(bx, NumSpec, i, j, k, n,
     {
-        R_store(i,j,k,n) = R_old(i,j,k,UFS+n)/state(i,j,k,URHO);
+        R_store(i,j,k,n) = R_old(i,j,k,UFS+n);
     });
 
+#if naux > 0
+    AMREX_PARALLEL_FOR_4D(bx, NumAux, i, j, k, n,
+    {
+        R_store(i,j,k,n) = R_old(i,j,k,UFX+n);
+    }
+#endif
+
     AMREX_PARALLEL_FOR_3D(bx, i, j, k,
     {
-        R_store(i,j,k,NumSpec) = R_old(i,j,k,UEDEN)/state(i,j,k,URHO);
-        R_store(i,j,k,NumSpec+1) = R_old(i,j,k,UEDEN);
-        R_store(i,j,k,NumSpec+2) = 0.0; // we're not storing the weights
+        R_store(i,j,k,NumSpec+NumAux  ) = R_old(i,j,k,UEDEN);
+        R_store(i,j,k,NumSpec+NumAux+1) = 0.0; // we're not storing the weights
     });
```

Comment:
are the weights no longer stored?

Reply:
That's not a change, you were already not storing the weights there (#1014).